src/main.py

```diff
@@ -1,18 +1,47 @@
 """
 Main project module
 """
-import yaml
+from console_utils import load_input_from_file
+from src.filters import remove_spaces, remove_repeats, charswap_filter, remove_endings
+from src.source_dicts import SourceDictionaries
 
-from console_utils import run_filters
-
-TEST_WORDS_FILEPATH = "../data/test_badwords.yaml"
+INPUT_FILE_PATH = "../wejscie.txt"
 
 if __name__ == '__main__':
-    user_input = input("Type word to check(skip if run words from test_badwords.yaml)\n")
-    if user_input == "":
-        with open(TEST_WORDS_FILEPATH, "r", encoding="utf8") as stream:
-            badwords = yaml.safe_load(stream)["badwords"]
-        for word in badwords:
-            run_filters(word=word)
+    option = input("1) Wczytaj tekst z pliku\n2) Wczytaj tekst z konsoli\n")
+    if option == "1":
+        text_to_filter = load_input_from_file(path=INPUT_FILE_PATH)
+    elif option == "2":
+        text_to_filter = input("Wprowadź tekst:\n")
     else:
-        run_filters(word=user_input)
+        option = input("Nieprawidłowa opcja\n1) Wczytaj tekst z pliku\n2) Wczytaj tekst z konsoli\n")
```

Comment:
Don't we need wail here?
```suggestion
good_input = false
while good_input:
    option = input("1) Wczytaj tekst z pliku\n2) Wczytaj tekst z konsoli\n")
    if option == "1":
        text_to_filter = load_input_from_file(path=INPUT_FILE_PATH)
        good_input = true
    elif option == "2":
        text_to_filter = input("Wprowadź tekst:\n")
        good_input = true
    else:
        run_filters(word=user_input)
        option = input("Nieprawidłowa opcja\n1) Wczytaj tekst z pliku\n2) Wczytaj tekst z konsoli\n")
```

Reply:
The line 22 works as one, but I think your solution is better

Reply:
you can change good_input to bad_input since its more understandable